src/auth.ts

```diff
@@ -40,23 +41,27 @@ export const authHandler = (type: 'login' | 'registration') => (
         methods: {
           password: {
             config: {
-              fields: {
-                csrf_token: { value: csrf_token = '' } = {},
-                identifier: { value: identifier = '' } = {},
-                'traits.email': { value: email = '' } = {},
-              } = {},
+              fields = {},
               action,
               errors,
             },
           },
         },
       } = request
 
+      // inject handlebars helper keys
+      const formFields = Object.values(fields).map<ExtendedFormField>(field => ({
+        ...field,
+        ...{
+          isHidden: field.type === 'hidden',
```

Comment:
As discussed, I think it would be better to move this to a partial with fallback